components/Dialogs/examples/DialogsWithEmphasisButtonExampleViewController.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+import MaterialComponents.MaterialButtons
+import MaterialComponents.MaterialDialogs
+import MaterialComponentsBeta.MaterialContainerScheme
+import MaterialComponentsBeta.MaterialDialogs_Theming
+
+class DialogsWithEmphasisButtonExampleViewController: UIViewController {
+
+  private let materialButton = MDCButton()
+  var containerScheme = MDCContainerScheme()
```

Comment:
It looks like the color/typography schemes assignments are missing. Is that intentional?

Reply:
It was intentional. I thought the theme of examples used by catalog are controlled by global injection. So assigning them here will cause confusion, because the color, typography may not match. Though I found in the typical example, it only assigns the color, which I think is confusing too.